shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -812,11 +815,14 @@ public void onTouchExplorationStateChanged(boolean enabled) {
             if (enabled) {
                 mTouchExplorationEnabled = true;
                 ensureAccessibilityEnabled();
+                nativeSetAssistiveTechnologyEnabled(mNativeView.get(), true);
             } else {
                 mTouchExplorationEnabled = false;
                 if (mAccessibilityNodeProvider != null) {
                     mAccessibilityNodeProvider.handleTouchExplorationExit();
                 }
+                nativeSetSemanticsEnabled(mNativeView.get(), false);
```

Comment:
Where is this set to true? Kinda expected that in the if branch?

Reply:
That is in ensureAccessibilityEnabled